examples/mplot3d/mixed_subplots.py

```diff
@@ -1,11 +1,10 @@
 """
-==================
-2D and 3D subplots
-==================
+============================================
+Demonstrate the mixing of 2D and 3D subplots
```

Comment:
I think the original title is more succinct and clear, but happy for it to change